Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -1694,11 +1696,19 @@ theorem coe_apply : coeAlgHom F f = f :=
   rfl
 #align ratfunc.coe_apply RatFunc.coe_apply
 
+theorem coe_coe (P : Polynomial F) : (P : LaurentSeries F) = (P : RatFunc F) := by
+  erw [RatFunc.coe_def, RatFunc.coeAlgHom, liftAlgHom_apply, RatFunc.num_algebraMap,
+    RatFunc.denom_algebraMap P, map_one, div_one]
+  rfl
```

Comment:
A proof without `erw`/`rfl` so better resistant against library changes:
```suggestion
  simp [RatFunc.coe_def]
```

Reply:
If you're speaking about the proof of `coe_coe`, the nice `simp [RatFunc.coe_def]` proof does not work for me...

Reply:
Hm, it worked for me... Maybe you also need to unfold the new `coePolynomial` definition in `simp`?

Reply:
Yep! Good catch.